### PR TITLE
filer.meta.backup: add -excludePaths flag to skip paths from backup

### PR DESCRIPTION
**Pull request by ai8future**
## Summary
Add a new `-excludePaths` flag to `weed filer.meta.backup` that accepts comma-separated path prefixes to exclude from backup operations.

This enables selective backup when certain directories (e.g., legacy buckets) should be skipped during both initial traversal and streaming backup.

## Changes
- Add `excludePaths` field to `FilerMetaBackupOptions` struct
- Add `-excludePaths` CLI flag
- Add `shouldExclude()` helper method to check path prefixes
- Filter excluded paths in `traverseMetadata()` (initial full sync)
- Filter excluded paths in `eachEntryFunc()` (streaming backup)

## Usage
```bash
weed filer.meta.backup -filerDir=/buckets   -excludePaths=/buckets/legacy1,/buckets/legacy2,/buckets/legacy3   -config=backup.toml
```

## Test plan
- [ ] Verify backup excludes specified paths during initial traversal
- [ ] Verify backup excludes specified paths during streaming updates
- [ ] Verify empty `-excludePaths` works (no filtering)
- [ ] Verify comma-separated multiple paths work correctly

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **New Features**
  * Added include- and exclude-prefix flags to control which absolute paths are backed up; prefixes are normalized and applied consistently.
  * Introduced a reusable path-prefix filter to evaluate inclusion rules.

* **Improvements**
  * Traversal and streaming now skip excluded paths, and renames avoid unnecessary create/delete when one side is excluded; logs show active include/exclude prefixes.

* **Tests**
  * Comprehensive unit tests cover include/exclude interactions, precedence, normalization, and edge cases.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>

### CODE REVIEW

File: weed/command/filer_meta_backup.go [hunk 1]
```diff
@@ -127,12 +129,28 @@ func (metaBackup *FilerMetaBackupOptions) initStore(v *viper.Viper) error {
...
+	if *metaBackup.excludePaths == "" {
+		return false
+	}
+	for _, excludePrefix := range strings.Split(*metaBackup.excludePaths, ",") {
+		if strings.HasPrefix(fullpath, excludePrefix) {
+			return true
+		}
+	}
+	return false
+}
```
[the remark this answers is not in the document]
Comment (ai8future) [reply weed/command/filer_meta_backup.go]: I submitted fixes for this

**Comment by ai8future** [comment]
Begging for an addition here. This will allow for people to phase-in their usage of filer.meta.backup. We've got a few postgres2 tables that are more than a billion rows so control over excluding certain buckets is fantastic.

**Comment by ai8future** [comment]
Thanks so much Chris!
